src/options.rs

```diff
@@ -355,6 +355,8 @@ pub enum Command {
     Instance(portable::options::ServerInstanceCommand),
     /// Manage local EdgeDB installations
     Server(portable::options::ServerCommand),
+    /// Manage local extensions
+    Extension(portable::options::ServerInstanceExtensionCommand),
```

Comment:
Does commands under `exension` always work some specific instance? If so, do you think it would fit better under the `instance` top-level command?

For example: `edgedb instance extension list-installed` and `edgedb instance extension install`.

Reply:
Per slack -- this was @elprans 's recommendation